### PR TITLE
Support pushing to Docker Hub

### PR DESCRIPTION
**Pull request by JohnNiang**
Please checkout the result from<https://github.com/JohnNiang/halo/runs/8225641705?check_suite_focus=true>.

/kind feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
    uses: halo-sigs/actions/halo-next-docker-build@main # prefer to specific ref.
    with:
      ghcr_token: ${{ secrets.GHCR_TOKEN }} # default is ""
+     dockerhub-user: halohub
+     dockerhub-token: ${{ secrets.DOCKERHUB_TOKEN }}
      push: true # default is false
      image-name: halodev # default is halo
```
Comment (ruibaby) [inline README.md:12]: 我们现在好像用的 halo-dev，admin 是 admin-dev

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by guqing — COMMENTED** [review]
/approve

**Pull request merged**